### DISCUSSION TITLE
How do i use the MultiSelectionComboBox?

### DISCUSSION THREAD

**Discussion by andrewccard** [MahApps/MahApps.Metro#4334] [Q&A] [answered]
There is documentation for it, but its not showing in the toolbox.

**Comment by timunie** [comment, marked as answer]
It will only be available in the next version, which should be released soon. For now you need to use the preview or wait a little bit.

**Reply by andrewccard** [reply]
How do i use the preview? Do I have to just build myself?

**Reply by timunie** [reply]
No I think there is a check box in the nuget explorer, which you have to mark.

**Reply by andrewccard** [reply]
There is no preview versions when I click the box.

**Reply by nekix** [reply]
Will a 3.0.0 preview be available on nuget? I used your MultiSelectionComboBox, but after reinstalling the system, the preview MahApps from nuget was lost...

**Reply by timunie** [reply]
Sorry for the late reply. Here is the nuget feed for latest alpha / beta versions of MahApps: https://github.com/MahApps/MahApps.Metro/wiki#appveyor-builds
